API/ev3_timer.c

```diff
@@ -85,52 +85,88 @@ void tmInitArrays()
 
 bool tmInitKernel()
 {
+    int retval;
     struct epoll_event event = {0};
     event.events = EPOLLIN;
 
     // initialize epoll
     tcb.waiterEpollFd = epoll_create(1);
-    RETURN_ON_FAIL(tcb.waiterEpollFd, "epoll_create1() failed", false);
+    if (tcb.waiterEpollFd < 0)
+    {
+        REPORT_ERRNO("epoll_create1() failed");

```

Comment:
Just use `perror`? `__FILE__` doesn't really hold any extra info.

Reply:
Resolved in 2f4cf12adfa6f569151b26578bd2cebfe06c2670 :+1: